docs/content/how-to/powervs/create-cluster-powervs.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Create PowerVS Hypershift Cluster
+---
+
+# Create PowerVS Hypershift Cluster
+
+Creating Hypershift cluster in IBM Cloud PowerVS service.
+
+## Prerequisites
+
+Please see [prerequisites](./prerequisites-and-env-guide.md/#prerequisites) before setting up the cluster
+
+## Creating the Cluster
+
+Use the `hypershift create cluster powervs` command:
+
+    ./bin/hypershift create infra powervs --base-domain BASEDOMAIN
+        --resource-group RESOURCE_GROUP \
+        --infra-id INFRA_ID \
```

Comment:
can we remove is this field isn't mandatory?

Reply:
removed